src/main/resources/templates/add.html

```diff
@@ -37,7 +37,7 @@ <h2>Open</h2>
     <span class="input-group-text" id="priceOpened">Price Opened</span>
     <input type="text" class="form-control" aria-label="Sizing example input" aria-describedby="inputGroup-sizing-sm" name="priceOpened" th:field="*{priceOpened}">
   </div>
-  <button name="submit" type="sumbit" value="0">Submit</button>
+  <button type="submit" name="submit" value="0">Submit</button>
```

Comment:
name and type were swapped, corrected typo from sumbit

ref: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3

Reply:
This was covered in a previous review.  When that occurs, first in wins.